core/Ballerina.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-org-name= "environmentplatform"
+org-name= "yashodperera"
```

Comment:
Revert to the previous name.
```suggestion
org-name= "environmentplatform"
```

Reply:
@deshankoswatte @yashodgayashan what do you think, shall we go ahead with just environmentplatform or any ideas for any other names like environmentplatformsl or something similar?

Reply:
It's a +1 from me fr anything which is suitable including what @sanjulamadurapperuma mentioned. @yashodgayashan thoughts?

Reply:
@yashodgayashan so shall we keep this as environmentplatform?